src/DIRAC/WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -392,3 +396,22 @@ def _checkPilotVersion(self, resourceDict):
                 "Version check requested but expected project %s != received %s"
                 % (validProject, resourceDict["ReleaseProject"])
             )
+
+
+def parseVersion(releaseVersion):
```

Comment:
Don't we have something very similar in the pilot repo and somewhere else ? 
How about factorizing it in the `diraccfg` repo ?

Reply:
We do indeed have something very similar for the Pilot, but in the Pilot we can only use "strictly python" as it's code that would run using the local python.